src/backend/aspen/api/config/config.py

```diff
@@ -205,45 +194,51 @@ def _AWS_SSM_PARAMETER(self, parameter_suffix) -> Mapping[str, Any]:
             parameter = get_parameter["Parameter"]["Value"]
             return json.loads(parameter)
 
-    @property
+    @cached_property
     def AWS_NEXTSTRAIN_SFN_PARAMETER(self) -> Mapping[str, Any]:
         return self._AWS_SSM_PARAMETER("nextstrain-ondemand-sfn")
 
     ####################################################################################
     # SFN runtime input properties
-    @property
+    @cached_property
     def NEXTSTRAIN_DOCKER_IMAGE_ID(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["Input"]["Run"]["docker_image_id"]
 
     ####################################################################################
     # SFN batch config properties
-    @property
+    @cached_property
     def NEXTSTRAIN_OUTPUT_PREFIX(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["OutputPrefix"]
 
-    @property
+    @cached_property
     def RUN_WDL_URI(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["RUN_WDL_URI"]
 
-    @property
+    @cached_property
     def NEXTSTRAIN_EC2_MEMORY(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["RunEC2Memory"]
 
-    @property
+    @cached_property
     def NEXTSTRAIN_EC2_VCPU(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["RunEC2Vcpu"]
 
-    @property
+    @cached_property
     def NEXTSTRAIN_SPOT_MEMORY(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["RunSPOTMemory"]
 
-    @property
+    @cached_property
     def NEXTSTRAIN_SPOT_VCPU(self) -> str:
         return self.AWS_NEXTSTRAIN_SFN_PARAMETER["RunSPOTVcpu"]
 
     class Config:
         env_file = ".env"
         env_file_encoding = "utf-8"
+        keep_untouched = (
+            cached_property,
+        )  # https://github.com/samuelcolvin/pydantic/issues/1241
 
 
-settings = Settings()
+@lru_cache()
```

Comment:
Even after googling I'm still unclear about the `@lru_cache` syntax - does it need to be called like a function, as you do here? Or are the parens not needed like in line 177?

Reply:
Used to be that we needed to call `lru_cache()` with parens to initialize a cache. Looks like as of python 3.8+ the parens are no longer required though! I'll remove the parens since I think we're using 3.9